pipeline/inputs/process-exporter-metrics.md

```diff
@@ -0,0 +1,84 @@
+---
+description: >-
+  A plugin based on Process Exporter to collect process level of metrics of system
+  metrics
+---
+
+# Process Exporter Metrics
+
+[Prometheus Node Exporter](https://github.com/prometheus/node_exporter) is a popular way to collect system level metrics from operating systems, such as CPU / Disk / Network / Process statistics. Fluent Bit 2.2 includes process exporter metrics plugin that builds off the Prometheus design to collect process level metrics without having to manage two separate processes or agents.
+
+The initial release of Process Exporter Metrics refers a process group level collector to collect metrics available from [the 3rd party implementation of Prometheus Process Exporter](https://github.com/ncabatoff/process-exporter) and we plan to expand them over time if needed.
+
+**Important note:** Metrics collected with Process Exporter Metrics flow through a separate pipeline from logs and current filters do not operate on top of metrics.
+
+This plugin is only supported on Linux based operating systems.
+
+
+## Configuration
+
+| Key                       | Description                                                                            | Default   |
+| ------------------------- | -------------------------------------------------------------------------------------- | --------- |
+| scrape_interval           | The rate at which metrics are collected from the host operating system                 | 5 seconds |
+| path.procfs               | The mount point used to collect process information and metrics                        | /proc/    |
```

Comment:
Any specification on the type of mount required, e.g. is read-only ok?

Reply:
Probably want an example for containers here - we cannot mount `/proc` so have to customise.

Reply:
Right. I added a note for read-only attribute: https://github.com/fluent/fluent-bit-docs/pull/1198/commits/d8e073872e2bb35794ff7847ed2984ce905fabb8